### ISSUE TITLE
Component not drawing over status bar anymore on iOS 13

### ISSUE THREAD

**Issue by ppamorim** [cezarywojcik/CWStatusBarNotification#116]
This component is not working anymore on iOS 13, I am trying to find a solution for this but can't see anything ideal to fix it.

Edit: The component will not work if your app uses `UIScene`.

**Comment by cculianu** [comment]
Yeah I have the same issue. I tried setting the .windowLevel on the internal notificationWindow instance that is used for the view -- to no avail.

It seems iOS 13 doesn't let you put any views or windows on top of the status bar!

I think this may be a bug in iOS 13.

I tried doing a quick "hide" of the status bar, then moving my app's window down 20 pix to "fake" it, then quickly unhide the status bar.  This looks ok on iPhone -- but this produces weird results on iPad in windowed and/or split-screen mode for me.

If anyone found a workaround, would love to hear it.

**Comment by ppamorim** [comment]
@cculianu Reported to Apple: https://forums.developer.apple.com/thread/124649?start=0&tstart=0

**Comment by cculianu** [comment]
Ok, thanks. I also added a backing set of screenshots and info to the post.  Who knows -- maybe Apple will fix this.  I consider this a bug in iOS, TBH.

Other projects that try to draw to the status bar area are also affected.  Most of them implement a workaround by just avoiding the area now on iOS13.

Sad.

**Comment by ppamorim** [comment]
@cculianu Mind that your comment is not visible, you will need to remove the link from them or fake it. Apple is not approving any comments with link...

**Comment by cculianu** [comment]
I put a link to the image...  Oh so I need to have the image url just be in the message but not as an `<img=` tag?

Why do they even enable HTML images then? LOL

It says "being moderated" -- I take it .. it will never get moderated?

**Comment by cculianu** [comment]
Ok. Fixed. Thanks for the tip.

**Comment by ppamorim** [comment]
Yes it never gets moderated...
